sidecar-injector/README.md

```diff
@@ -13,101 +31,199 @@ The result should be:
+
+Create a namespace "cyberark-sidecar-injector", where you will deploy the CyberArk Sidecar Injector Webhook components.
 
 1. Create namespace
     ```bash
-    ~$ kubectl create namespace secretless
+    ~$ kubectl create namespace cyberark-sidecar-injector
     ```
 
-## Deploy
+### Deploy Sidecar Injector
 
 1. Create a signed cert/key pair and store it in a Kubernetes `secret` that will be consumed by sidecar deployment
     ```bash
     ~$ ./deployment/webhook-create-signed-cert.sh \
-        --service secretless-sidecar-injector-webhook-svc \
-        --secret secretless-sidecar-injector-webhook-certs \
-        --namespace secretless
+        --service cyberark-sidecar-injector \
+        --secret cyberark-sidecar-injector \
+        --namespace cyberark-sidecar-injector
     ```
 
-2. Patch the `MutatingWebhookConfiguration` by set `caBundle` with correct value from Kubernetes cluster
+2. Patch the `MutatingWebhookConfiguration` by setting `caBundle` with correct value from Kubernetes cluster
     ```bash
     ~$ cat deployment/mutatingwebhook.yaml | \
         deployment/webhook-patch-ca-bundle.sh \
-          --service secretless-sidecar-injector-webhook-svc \
-          --namespace secretless > \
+          --service cyberark-sidecar-injector \
+          --namespace cyberark-sidecar-injector > \
         deployment/mutatingwebhook-ca-bundle.yaml
     ```
 
 3. Deploy resources
     ```bash
-    ~$ kubectl -n secretless apply -f deployment/deployment.yaml
-    ~$ kubectl -n secretless apply -f deployment/service.yaml
-    ~$ kubectl -n secretless apply -f deployment/mutatingwebhook-ca-bundle.yaml
+    ~$ kubectl -n cyberark-sidecar-injector apply -f deployment/deployment.yaml
+    ~$ kubectl -n cyberark-sidecar-injector apply -f deployment/service.yaml
+    ~$ kubectl -n cyberark-sidecar-injector apply -f deployment/mutatingwebhook-ca-bundle.yaml
     ```
 
-## Verify
+### Verify Sidecar Injector Installation
 
 1. The sidecar injector webhook should be running
     ```bash
-    ~$ kubectl -n secretless get pods
+    ~$ kubectl -n cyberark-sidecar-injector get pods
     ```
     ```
     NAME                                                  READY     STATUS    RESTARTS   AGE
-    secretless-sidecar-injector-webhook-deployment-bbb689d69-882dd   1/1       Running   0          5m
+    cyberark-sidecar-injector-bbb689d69-882dd   1/1       Running   0          5m
     ```
     ```bash
-    ~$ kubectl -n secretless get deployment
+    ~$ kubectl -n cyberark-sidecar-injector get deployment
     ```
     ```
     NAME                                  DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-    secretless-sidecar-injector-webhook-deployment   1         1         1            1           5m
+    cyberark-sidecar-injector             1         1         1            1           5m
     ```
 
-## Deploy test application
+## Installing the Sidecar Injector (Helm)
+
++ [Helm](https://docs.helm.sh/using_helm/) is **required**
```

Comment:
Just out of curiosity, what does Helm provide us that we couldn't do on our own here?

Reply:
packages up the whole experience in 1 go. easier deployment UX